_episodes/04-changes.md

```diff
@@ -170,60 +148,67 @@ nothing to commit, working directory clean
 ~~~
 {: .output}
 
-it tells us everything is up to date.
-If we want to know what we've done recently,
-we can ask Git to show us the project's history using `git log`:
+it tells us everything is up to date.  If we want to know what we've
+done recently, we can ask Git to show us the project's history using
+`git log`:
 
 ~~~
 $ git log
 ~~~
 {: .bash}
 
 ~~~
-commit f22b25e3233b4645dabd0d81e651fe074bd8e73b
-Author: Vlad Dracula <vlad@tran.sylvan.ia>
-Date:   Thu Aug 22 09:51:46 2013 -0400
+commit c118dee0fd8f6d8733ceb10b90ce6431eedd47e4 (HEAD -> master)
+Author: Alex Dunn <dunn.alex@gmail.com>
```

Comment:
Just want to clarify that you're cool with having your name/email throughout the lessons.

Reply:
I mean it's already all over my git commits, so sure.